timbermill-client/src/main/java/com/datorama/oss/timbermill/unit/EventsList.java

```diff
@@ -0,0 +1,17 @@
+package com.datorama.oss.timbermill.unit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EventsList extends ArrayList<Event> {
+
+    public EventsList(List<Event> events) {
+        super(events);
+    }
```

Comment:
the super create a copy of each event and store it in itself (arraylist)
do you think it's better to use the original events?

Reply:
@edengal also want to make sure the specific implementation (arrayList) doesn't bother you,
I didn't find a way to do that with just List

Reply:
Yeah, let's extend LinkedList instead so we will not copy
I'm ok with this implementation